node/src/chain_spec.rs

```diff
@@ -269,12 +305,24 @@ fn testnet_genesis(
                 })
                 .collect::<Vec<_>>(),
         }),
+
+        // TODO
+        // Create two tokens and corresponding pool through xyk for the authority account and pass it in here.
         pallet_staking: Some(StakingConfig {
             validator_count: initial_authorities.len() as u32 * 2,
             minimum_validator_count: initial_authorities.len() as u32,
-            stakers: initial_authorities
+            stakers: staking_accounts
                 .iter()
-                .map(|x| (x.2.clone(), x.2.clone(), 0_u128, StakerStatus::Validator))
+                .map(|x| {
+                    let (account_id, _, _, _, _, liquidity_token_id, liquidity_token_amount) = x;
```

Comment:
can  `(account_id, _, _, _, _, liquidity_token_id, liquidity_token_amount)`  be used directly in map statement ? :thinking: 

```
map(|(account_id, _, _, _, _, liquidity_token_id, liquidity_token_amount)|{
```

Reply:
oh yeah, lol... I'll keep it in mind.
Lots of cases in chain_spec, I'll make a mini-task in trello to fix this.